### ISSUE TITLE
Can you tell me meaning of loss values?

### ISSUE THREAD

**Issue by richard0326** [andy-yun/pytorch-0.4-yolov3#75]
if 
`Layer(106) nGT   2, nRC   1, nRC75   0, nPP   7, loss: box  1.454, conf  0.942, class  0.152, total   2.548`

is printed...
What does Layer, nGT, nRC, nRC75, nPP means?

**Comment by andy-yun** [comment]
@richard0326 
nGT: number of ground truth
nRC: number of recall > 0.5
nRC75: number of recall > 0.75
nPP: number of proposals

**Comment by richard0326** [comment]
What does Layer mean in this training??

**Comment by richard0326** [comment]
Layer ( n ) 괄호 안에 있는 n의 의미가 뭔가요?

**Comment by andy-yun** [comment]
@richard0326 Layer(n) means the nth layer in the yolo v3 structure. you can easily find the yolo layers in the structure. n means the position of [yolo] layer.

**Comment by richard0326** [comment]
Thank a lot!

**Comment by richard0326** [comment]
I have one more question

8: Layer(082) nGT  27, nRC   4, nRC75   2, nPP   0, loss: box 128.327, conf  8.406, class 19.071, total 155.805
8: Layer(094) nGT  27, nRC  11, nRC75   2, nPP 16224, loss: box  8.211, conf 1075.647, class 23.411, total 1107.269
8: Layer(106) nGT  27, nRC   0, nRC75   0, nPP 64896, loss: box 21.769, conf 5427.244, class 23.114, total 5472.127

in training I get 3 layer result for each step...
Why do I have 3 output for one training??

is it just output of the middle layer?
